code/modules/food_and_drinks/pizzabox.dm

```diff
@@ -270,56 +286,55 @@
 
 /obj/item/pizzabox/bomb/Initialize(mapload)
 	. = ..()
-	var/randompizza = pick(subtypesof(/obj/item/food/pizza))
-	pizza = new randompizza(src)
+	if(!pizza)
+		var/randompizza = pick(subtypesof(/obj/item/food/pizza))
+		pizza = new randompizza(src)
 	bomb = new(src)
 	bomb.installed = TRUE
 	wires = new /datum/wires/explosive/pizza(src)
 
-/obj/item/pizzabox/margherita/Initialize(mapload)
-	. = ..()
-	AddPizza()
-	boxtag = "Margherita Deluxe"
+/obj/item/pizzabox/bomb/armed
+	bomb_timer = 5
+	bomb_defused = FALSE
+	boxtag = "Meat Explosion"
```

Comment:
This should not be discernable to players.

Reply:
make it a suggestion so its easier to commit mate